tests/unit/Logger.js

```diff
@@ -8,8 +8,8 @@ const loggingMisuseGenerator = Utils.loggingMisuseGenerator;
 const DummyLogger = Utils.DummyLogger;
 
 const Config = require('../../lib/Config.js');
+const Logger = require('../../lib/Logger.js');
 const RequestLogger = require('../../lib/RequestLogger.js');
-const Logger = require('../../index.js').Logger;
```

Comment:
I first move it for the alphabetical order, and then I wanted to make it looks like others.